interfaces/cython/cantera/liquidvapor.py

```diff
@@ -4,47 +4,180 @@
 from . import PureFluid, _cantera
 
 
-def Water():
+def Water(backend='default'):
```

Comment:
Perhaps `Reynolds` would be a better name for this backend?

Reply:
(although opting for lower-case)